Documentation/install/upgrade.rst

```diff
@@ -316,6 +316,24 @@ Annotations:
 1.7 Upgrade Notes
 -----------------
 
+1.7.13 Upgrade Notes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+A new flag ``--egress-multi-home-ip-rule-compat`` was introduced to allow
+specifying whether to maintain compatibility with the older ENI mode datapath.
+It defaults to ``false`` meaning it will instruct Cilium to **not** maintain
+compatibility and perform a migration to the new ENI datapath, upon Cilium
+upgrade. This operation will modify the routing table to fix a potential
+collision.
+
+If the flag is ``true`` upon upgrade, Cilium will **not** perform a migration.
+
+If the migration was already completed and a downgrade to an older version of
+Cilium is desired, you must first set the flag to ``true`` and then restart
+Cilium so that it can rollback the migration. This ensures that the new ENI
+datapath is rolled back to a state where it is compatible with the older
+version of Cilium.
+
 .. _current_release_required_changes:
 
 .. _1.7_required_changes:
```

Comment:
nit: Should probably make these link targets above the new section so that users don't accidentally skip over these notes when upgrading.

We can also call this out for ENI users in the release notes.